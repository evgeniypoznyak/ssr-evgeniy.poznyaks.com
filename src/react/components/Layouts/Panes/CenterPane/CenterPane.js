import React from 'react';
import {Grid, Paper} from '@material-ui/core';

const CenterPane = props => {
    return (
        <Grid item lg={8} md={8} sm={12} xs={12}>
            <Paper style={props.styles.CenterPane}>
                {props.children}
            </Paper>
        </Grid>
    );
};

export default CenterPane;
