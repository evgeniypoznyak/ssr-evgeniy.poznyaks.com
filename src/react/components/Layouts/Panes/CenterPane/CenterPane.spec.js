import React from 'react';
import {mount} from 'enzyme';
import CenterPane from './CenterPane';

let wrapper;
const styles = {
    CenterPane: {
        height: '100%',
        padding: 20,
        marginTop: 5,
        marginBottom: 10,
        marginLeft: 10,
        marginRight: 10,
    },
};

beforeAll(() => {
    wrapper = mount(<CenterPane styles={styles}>test</CenterPane>);
});

describe('CenterPane', () => {
    it('should render correctly with enzyme mount', () => {
        expect(wrapper.children.length).toBe(1);
        expect(wrapper.getElement()).toMatchSnapshot();
    });
    it('should be text test on the page', () => {
        expect(wrapper.text()).toMatch('test');
    });
});
