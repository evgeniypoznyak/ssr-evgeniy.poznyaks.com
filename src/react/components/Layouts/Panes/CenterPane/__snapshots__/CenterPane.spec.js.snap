// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterPane should render correctly with enzyme mount 1`] = `
<CenterPane
  styles={
    Object {
      "CenterPane": Object {
        "height": "100%",
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 5,
        "padding": 20,
      },
    }
  }
>
  test
</CenterPane>
`;
